maintainers/maintainer-list.nix

```diff
@@ -2927,6 +2927,16 @@
     githubId = 147284;
     name = "Jason Felice";
   };
+  erdnaxe = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add erdnaxe`` and sort it alphabetically.